projects/storefrontlib/cms-components/content/video/video.component.html

```diff
@@ -0,0 +1,25 @@
+<div
+  *ngIf="data$ | async as data; else loading"
+  class="video-container"
+  tabindex="-1"
```

Comment:
what it is used for?

Reply:
I'm not which one you're specifically referring to but:

Div - Used as container to help with positioning elements especially overlayTitle
loading - Used to display spinner if data is not ready (I saw this in other CMS components)
tabindex="-1" - Used to not making the div tabable so it is more seemless. (You would need to tab twice to get to the video element before, now it's just once)